libcontainer/cgroups/fs/fs.go

```diff
@@ -46,9 +29,10 @@ type subsystem interface {
 }
 
 type manager struct {
-	mu      sync.Mutex
-	cgroups *configs.Cgroup
-	paths   map[string]string
+	mu         sync.Mutex
+	cgroups    *configs.Cgroup
+	paths      map[string]string
+	subsystems []subsystem
```

Comment:
It does not make sense to have a per-manager subsystem list.

Instead, I think, we should keep subsystems global, and wrap their initialization into `once.Do`.